src/main/webapp/WEB-INF/views/car/all.jsp

```diff
@@ -0,0 +1,37 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>All cars:</title>
+</head>
+<body>
+<table>
+  <tr>
+    <td>Id</td>
+    <td>Model</td>
+    <td>Manufacturer name</td>
+  </tr>
+  <c:forEach items="${cars}" var="car">
+  <tr>
+    <td><c:out value="${car.id}"/></td>
+    <td><c:out value="${car.model}"/></td>
+    <td><c:out value="${car.manufacturer == null ? 'NULL' : car.manufacturer.name}"/></td>
+    <td><a href="${pageContext.request.contextPath}/car/delete?id=${car.id}">delete</a> </td>
+  </tr>
+  <tr>
+    <td>Drivers</td>
+    <td>id</td>
+    <td>name</td>
+    <td>license number</td>
+  </tr>
+  <c:forEach items="${car.drivers}" var="driver">
+  <tr>
+    <td></td>
+    <td><c:out value="${driver.id}"/></td>
+    <td><c:out value="${driver.name}"/></td>
+    <td><c:out value="${driver.licenseNumber}"/></td>
+  </tr>
```

Comment:
please, provide a screenshot with this table

Reply:
<img width="428" alt="Screenshot 2022-11-23 at 14 28 32" src="https://user-images.githubusercontent.com/92673432/203548406-213df15a-85af-47dc-b003-e8c4e3c74b6e.png">